epysurv/models/timepoint/_base.py

```diff
@@ -103,7 +103,7 @@ def _get_start_epoch(data: pd.DataFrame) -> int:
 class SurveillanceRPackageAlgorithm(TimepointSurveillanceAlgorithm):
     """Base class for the algorithm from the R package surveillance."""
 
-    def predict(self, data: pd.DataFrame) -> pd.DataFrame:
+    def predict(self, data: pd.DataFrame, get_alarm_only : bool = True) -> pd.DataFrame:
```

Comment:
Great point on these.  I just committed the suggested change for the documentation string and the typo I had.

Reply:
Also, I have now updated this so that the previous `get_alarm_only` flag is not part of the API, but instead added a check to see if `upperbound` is part of the prediction frame and return it if it does exist.

Reply:
OK, I fixed up the API recommendation so that no flag is used anymore and resolved the formatting issues as well (via `black` and `isort`)